tutorials/Foyer-tutorials.ipynb

```diff
@@ -83,78 +184,915 @@
+      "  warnings.warn(msg)\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308912576>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308910080>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308912960>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308912240>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308911952>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308909792>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308910560>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308911040>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308911088>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308884640>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308888288>  detected\n",
+      "  warnings.warn(\n",
+      "/Users/cri/miniforge3/envs/mosdef-workshop/lib/python3.9/site-packages/gmso/core/topology.py:549: UserWarning: Non-parametrized Connection <Improper Improper, id: 6308884880>  detected\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "from foyer import general_forcefield\n",
```

Comment:
You may want to mention in this case the ability to convert a foyer forcefield (where potential forms are implied) to a GMSO forcefield where they're explicit. `gmso.core.forcefield.ForceField` is the relevant class. I think we're going to start to move away from the general_forcefield.

Reply:
I was actually thinking about that, but didn't want to get unfocused since we wouldn't have talked about the gmso format yet.  Might be good for you to refer back to this in your GMSO presentation.

Reply:
I'll add that to the GMSO tutorials PR #5.